services/brig/src/Brig/Data/Client.hs

```diff
@@ -233,6 +244,9 @@ insertClient = "INSERT INTO clients (user, client, tstamp, type, label, class, c
 updateClientLabelQuery :: PrepQuery W (Maybe Text, UserId, ClientId) ()
 updateClientLabelQuery = "UPDATE clients SET label = ? WHERE user = ? AND client = ?"
 
+updateClientSupportedFeaturesQuery :: PrepQuery W (Maybe [SupportedClientFeature], UserId, ClientId) ()
```

Comment:
This should be a cassandra set (`C.Set` from  `ìmport Cassandra as C`) and not a Maybe List. See e.g. https://github.com/wireapp/wire-server/blob/develop/services/galley/src/Galley/Data/Queries.hs#L206-L207